test/modules/DirectMethodSender/Program.cs

```diff
@@ -28,7 +28,7 @@ public static async Task<int> MainAsync()
             {
                 Guid batchId = Guid.NewGuid();
                 Logger.LogInformation($"Batch Id={batchId}");
-                Logger.LogInformation($"Load gen delay start for {Settings.Current.TestStartDelay}.");
+                Logger.LogInformation($"Direct Method Sender delay start for {Settings.Current.TestStartDelay}.");
```

Comment:
please send out another PR to fix this line.

Reply:
Type.